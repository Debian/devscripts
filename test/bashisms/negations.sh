#!/bin/sh

case "moo" in
    [^f]oo) # BASHISM
	echo hey
	;;
    [!f]oo)
	echo hey
	;;
esac

