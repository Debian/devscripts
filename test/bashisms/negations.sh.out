possible bashism in bashisms/negations.sh line 4 ([^] should be [!]):
    [^f]oo) # BASHISM
